install/update.rst

```diff
@@ -101,7 +99,7 @@ Updating Zammad
 
       Step 1: Ensure dependencies
          Before proceeding, double-check that your system environment matches
-         :doc:`Zammad’s requirements </prerequisites/software>`.
+         :doc:`Zammad's requirements </prerequisites/software>`.
```

Comment:
(only partially related) There is a 'danger' box at the start of the source update tab. I guess we don't need that any more, that change was quite a while ago.